### PR TITLE
Loadout adjustments.

### PR DESCRIPTION
**Pull request by Adrian16199**
## About The Pull Request
Removes these items from loadouts:
First aid kit
Frontier medical kit
Civil defense medical kit
Standart power cell
(While making this pr, i realized that the full frontier kit in itself costs 800 credits while things inside cost 660 and having just that from roundstart feels insane tbh)
## Why It's Good For The Game
Anyone using these items would pretty much have advantage compared to other players who would choose to go for more roleplay-ish items from roundstart. Especially with the medkits as it pretty much allows them to skip certain things like invading medical to obtain one or they save up money as they dont have to buy medicine.

## Proof Of Testing
<details>
<summary>Screenshots/Videos</summary>

</details>

## Changelog
:cl:
balance: Removes medkits from loadouts.
/:cl:

### CODE REVIEW

**Comment by Strazyplus** [comment]
it was powercreep, i'd say is but we had these items for a while. now why i'd think their removal would be bad. is the stations just don't naturally have enough of these items on the station and I prefer these items being abundant than scarce on round start. 

Before these items where in the loadout:
Mad-dash for tool storage to get tools, mainly crowbars, vending machine got emptied within the first 10 minutes sometimes.
Mad dash to take the 1-2 medkits in medbay lobby, and woe is you who was too slow.
etc, etc.

i don't want to have to run around the entire station just to grab a medkit and a crowbar at round start like a powergaming loot goblin.

**Comment by Adrian16199** [comment]
My question is, do you need those medkits?
Lets say theoraticly, you are an officer so you may actualy need them.
Whats stopping you from getting them from cargo or perchance, asking the bored chemist for a hypokit?
There are times where people tell me: Why do people not ask for medicine.
Because they can just start with it.
When it comes to the crowbar.... I can understand since in an emergency, you will need a crowbar to mayhaps not die a horrible death. Im not neccesarily against the toolbox but the medkits feel a bit like.... too much.
Im mostly doing these things just so you know... you interact with chemists or medical to obtain medicine or go to botany and either ask or get cloth from the biogenerator. Give meaning to some jobs more.

**Review by Majkl-J — CHANGES_REQUESTED** [review]
The soap can definetely stay. As silly as it is with the slipping it's just a worse cloth rag in every other way.

**Comment by Adrian16199** [comment]
> The soap can definetely stay. As silly as it is with the slipping it's just a worse cloth rag in every other way.

Wait, is it? I would have assumed other way around tbh but sure, dont see why not.

**Comment by Adrian16199** [comment]
Hum, what if, instead of entire toolbox, just a crowbar and sure, maybe lets keep the extinguisher.

**Comment by Arturlang** [comment]
@Adrian16199 There is the emergency toolbox that is pretty much what you want

**Comment by Adrian16199** [comment]
Fair enough~

**Comment by theselfish** [comment]
> it was powercreep, i'd say is but we had these items for a while. now why i'd think their removal would be bad. is the stations just don't naturally have enough of these items on the station and I prefer these items being abundant than scarce on round start.

Scarcity is the point.

**Comment by theselfish** [comment]
I don't see the harm in the cloth, honestly. Other than makeshift welding goggles, which you can easily make by cutting a dorm sheet up anyway.

**Comment by Formerxboxgamer** [comment]
Give us an empty version of the frontier kit I like the way it looks on my belt. It’s a cooler looking box for my medical beakers

**Comment by Adrian16199** [comment]
Hm. *Hm*.

**Comment by Adrian16199** [comment]
image
Two good news, i figured out why i was never able to turn on my own version of byond
Second good news, i made that proposition of an empty frontier kit to be a thing.
Very cool.

**Comment by NanoCats** [comment]
I'd like to speak against implementation of this PR, while I fully understand wanting to reduce abundance and encourage roleplay. There are significant drawbacks left unaddressed that could negatively impact player experience.

Removing the medkits will create an undue shortage of them, because at peak hours we get player counts of well over 100 (114 connected clients at time of writing). You're never going to be able to find a medkit ever again. God help you if you latejoin.
Also synths have it hard enough to the point hat I don't think letting them have one shitty power cell is worth removing.
There are other things like the fact that tools are incredibly important in this game overall, and cargo already suffers enough with half the station walking up to the desk like "Can I have a multitool please miss🥺 "

Why haven't you removed the pizza if that's how we're rolling?

All this stuff sort of, pales in comparison to items you can get by playing the game anyways, I laud the person who added these items to loadouts because "go find this relatively basic item and put it in your bag" wasn't exciting, dynamic gameplay I desperately needed in my every-shift gameplay loop.  There's also a power balance thing where people who have roles who can just get these items mid-round can easily access them and others can't. 

TL;DR it's just going to grief a huge number of people who don't deserve it in a not insignificant way.

**Comment by Adrian16199** [comment]
> I'd like to speak against implementation of this PR, while I fully understand wanting to reduce abundance and encourage roleplay. There are significant drawbacks left unaddressed that could negatively impact player experience.
> 
> Removing the medkits will create an undue shortage of them, because at peak hours we get player counts of well over 100 (114 connected clients at time of writing). You're never going to be able to find a medkit ever again. God help you if you latejoin. Also synths have it hard enough to the point hat I don't think letting them have one shitty power cell is worth removing. There are other things like the fact that tools are incredibly important in this game overall, and cargo already suffers enough with half the station walking up to the desk like "Can I have a multitool please miss🥺 "
> 
> Why haven't you removed the pizza if that's how we're rolling?
> 
> All this stuff sort of, pales in comparison to items you can get by playing the game anyways, I laud the person who added these items to loadouts because "go find this relatively basic item and put it in your bag" wasn't exciting, dynamic gameplay I desperately needed in my every-shift gameplay loop. There's also a power balance thing where people who have roles who can just get these items mid-round can easily access them and others can't.
> 
> TL;DR it's just going to grief a huge number of people who don't deserve it in a not insignificant way.

I mean, i fully understand your point, especially if the prices for the medical equipment is completely tragic like 800 credits for a mediocre first aid kit is pretty depressing tbh.
Im mostly making this PR as i have felt that it felt alot better when i wasnt on roundstart with a medkit pack on other servers, having to think of other ways than to just conveniently heal myself was pretty entertaining.
Plus this pr is at the point where it solely removes medical kits. My issue is tho, yes, 100 players, but do those 100 players play the game of being an antag or security, do they constantly fight to show a reason for such need of a medkit or is it just, just *incase*.
I agree with the tools since yeah, i guess it would be pretty annoying for cargo.
However i dont understand one thing.
How does the power cell affect synths.
I would get if you said etherals since you know, you can shlorp up batteries but... synths? 🤔 
(its an extremely pathetic cell like, 10mj, its not even gonna last 1 minute if you would put it into an entombed suit.)
But even then, despite everything. This PR could be taken down and I will be completely already satisfied with the results as i have learnd opinions regarding even such a simple thing on top of being bit better in coding.
Take that as you will. Plus I dont like that powercell as it made me think its the 100mj one but in reality its the 10mj that is literal crime to even exist for gaslighting me with how bad it is.

**Comment by Adrian16199** [comment]
But even then, if we wouldnt be removing these medkits for such reasons or many others.
It would be atleast a good thing to atleast tell me to add synth kits then.
Would be bit odd how flesh gets to have 3 medkits while synths get none roundstart
*Would be bit funny*, just my opinion tho.

**Comment by sippykot** [comment]
please don't... why is this needed? :/
it might be "more balanced" but - is the current state of things actually having an adverse effect on anything?

**Comment by sippykot** [comment]
> But even then, if we wouldnt be removing these medkits for such reasons or many others. It would be atleast a good thing to atleast tell me to add synth kits then. Would be bit odd how flesh gets to have 3 medkits while synths get none roundstart _Would be bit funny_, just my opinion tho.

i am in favour of adding the basic synth kit to the loadout honestly, synth healing above slapping someone with a welder/wires is quite rare

**Comment by NargaInDisguise** [comment]
This is such an absolutely pointless nerf after we've had them in for so long. I am incredibly vocally against this.
God forbid we can bring anything useful on station that you can already get round-start if you just rush where they spawn.

**Comment by NargaInDisguise** [comment]
Addition addendum: 

Instead of taking the nuclear option of "delete this it bad," take a more approachable option, such as allowing any dangerous job or medical job to round-start a medkit, such as miners and medical.

**Comment by NargaInDisguise** [comment]
Remaking this into a single comment:

This is a nerf that will (somewhat rightfully) recieve a lot of backlash. Many players enjoy having their round-start tools, and I personally can't blame them. I'm mostly against this change because I enjoy streamlining the somewhat boring early game for jobs that generally need to get up off the ground immediately (miner).

So heres my re-worked proposal: Instead of a nuclear "delete this it bad," add job locks to different items. Medkits? Let any dangerous jobs + medical start with them. Toolboxes? Any job that regularly has to build things, I.E. Science + Engineering. A basic powercell is just, such an inconsequential thing to have, honestly.

The frontier medical kits are fucking ridiculous and never should've been added to begin with, have you seen what some of those medpens are filled with? Those can go 100%.

**Comment by StrangeWeirdKitten** [comment]
The jobs that need these items already spawn with said items, adding a lock would be really redundent versus things that are deposited inside their job gear. Especially mining, the survival pen and the cheap brute kits are plenty for their work.

Overall this is targetted to removing the medkits from the general loadout. I understand removing the nice things like this is not something that anyone likes, but I also don't enjoy the gradual power creep in the loadout throughout the years as it's pretty much long violated keeping things that are mostly role play or cosmetic focus or their mechanics are very non-impactful that it doesn't really matter. Med kits are not roleplay or cosmetic focus.

Job gear is already given at spawn, and it's not medical doctors taking medkits from the loadout. The same with engineers already spawning with fully equipped toolbelts or robotics with synth repair kits. It'll be very redundent to then lock it to specifically the jobs themselves.

**Review by StrangeWeirdKitten — APPROVED** [review]

**Comment by Adrian16199** [comment]
Well okay but at some point, i have kept the tools, plus i mentioned that i did.
This pr has devolved into purely the medkits thing and that one powercell that is 10mj
I would be more willing to replace it with 100mj so people dont get gaslighted into thinking they are getting an actual powercell compared to the powercell that shouldnt even exist in my humble opinion.

**Comment by Hemlock32** [comment]
> > But even then, if we wouldnt be removing these medkits for such reasons or many others. It would be atleast a good thing to atleast tell me to add synth kits then. Would be bit odd how flesh gets to have 3 medkits while synths get none roundstart _Would be bit funny_, just my opinion tho.
> 
> i am in favour of adding the basic synth kit to the loadout honestly, synth healing above slapping someone with a welder/wires is quite rare

This infact already is the case on a certain other server

**Comment by Strazyplus** [comment]
Man this PR got gutted.... I perfectly understand this PR (in its original state) in a balance mindset to be absolutely correct however always was going to be a completely unpopular change.

I think things should be focused on fun instead of balance. but I do not believe abandoning balance like a certain other server is the way to go. while out of scope of this pr I think the correct solution is to have expensive items have a cost to them. which reflects your bank balance at start of the game. (meaning you can get everything like normal but if you grab all the powercreep expensive items and put yourself into the red, you get a debt moodlet until your bank balance goes positive.

**Comment by Adrian16199** [comment]
> Man this PR got gutted.... I perfectly understand this PR (in its original state) in a balance mindset to be absolutely correct however always was going to be a completely unpopular change.
> 
> I think things should be focused on fun instead of balance. but I do not believe abandoning balance like a certain other server is the way to go. while out of scope of this pr I think the correct solution is to have expensive items have a cost to them. which reflects your bank balance at start of the game. (meaning you can get everything like normal but if you grab all the powercreep expensive items and put yourself into the red, you get a debt moodlet until your bank balance goes positive.
> 
> **Adrian spite making a unpopular PR, you have communicated, took feedback and made changes to it. for that I appreciate you, marks of a good Contributor.**

I definitely like your idea.
My issue is that I have absolutely no idea on how would one do it. Could try. *Someday*, would definitely put alot of people into debt since it would *possibly* take every single item's price into consideration and latejoining you only have like.... 300 credits at worst and each piece of clothing usually costs 50 so i feel like said system would only need to consider pocket items or inhand items.
Also to the uh... praise.
Well... thank you. I dont think i should say anything else but, thank you for saying it, even if you didnt had to.

**Comment by Formerxboxgamer** [comment]
> Hm. _Hm_.

Goated TY

**Review by Majkl-J — APPROVED** [review]

**Comment by sippykot** [comment]
uhhh... why was this merged?

**Comment by NargaInDisguise** [comment]
WOAH woah woah, I don't believe this should've been merged like this. I strongly believe the roundstart basic medkit was more than fine. Literally only the deforest things should've gone, those were undoubtably powercreep.

**Comment by Razurath** [comment]
And all of mining, security, engineering, and medical cried because someone merged this. Seriously why was this implemented I've yet to see a good reason from anyone being "the rp" when all it's doing is creating artificial scarcity of healing items in a very dangerous game.

**Comment by Majkl-J** [comment]
> And all of mining, security, engineering, and medical cried because someone merged this. Seriously why was this implemented I've yet to see a good reason from anyone being "the rp" when all it's doing is creating artificial scarcity of healing items in a very dangerous game.

[Healing with no loss of resources]https://tgstation13.org/wiki/Surgery#Tend_Wounds
[Make your own healing](https://tgstation13.org/wiki/Guide_to_chemistry)
[800cr for a whole bunch of medical items](https://tgstation13.org/wiki/Supply_crates#Medical)
[First-Aid kit for 150cr wowza!](https://tgstation13.org/wiki/Supply_crates#Goodies)

[Alternative source of healing in case somehow you can't pull off the rest](https://tgstation13.org/wiki/Guide_to_hydroponics#Ambrosia_deus)

**Comment by Razurath** [comment]
> > And all of mining, security, engineering, and medical cried because someone merged this. Seriously why was this implemented I've yet to see a good reason from anyone being "the rp" when all it's doing is creating artificial scarcity of healing items in a very dangerous game.
> 
> [Healing with no loss of resources](https://tgstation13.org/wiki/Surgery#Tend_Wounds) [Make your own healing](https://tgstation13.org/wiki/Guide_to_chemistry) [800cr for a whole bunch of medical items](https://tgstation13.org/wiki/Supply_crates#Medical) [First-Aid kit for 150cr wowza!](https://tgstation13.org/wiki/Supply_crates#Goodies)
> 
> [Alternative source of healing in case somehow you can't pull off the rest](https://tgstation13.org/wiki/Guide_to_hydroponics#Ambrosia_deus)

Cool. When I start a mining shift with 800 credits I drop all of that on additional healing supplies and equipment that science can't give me right away so I can get my job going faster and hit vents better. As things are, starting with one medkit saves me 5-10 minutes I'd have to spend not doing the job I enjoy in the game. Because I don't want to wait on chemistry, I can't always get surgery, neither I nor cargo can afford much round start, and the station needs resources. There's usually one or two medkits in cargo, but if a tech grabs one and another miner grabs the other, others who need them are screwed until more time has passed in the shift. It's really neat to link to me to things I clearly didn't know, like that we can heal with surgery, grow plants, and make healing chems. However none of those help when you're a sec officer on the far side of the station trying desperately not to die or to let someone else die and for some reason one person decided you're not allowed to have a medkit anymore. Or the three person team responsible for providing the station its resources need to waste more time scrouging for it.

What GOOD does this PR bring to the game for us?

**Comment by Arturlang** [comment]
I think taking the cost out of your bank account balance as a sort of debt for priced loadout items is worth considering, if we want to re-add this

**Comment by Razurath** [comment]
Doesn't answer my question of what good it brings to the game.

**Comment by Majkl-J** [comment]
> What GOOD does this PR bring to the game for us?

Giving other people a job they're meant to be doing anyhow. Nothing stops you, as the sec officer, from asking medbay for chems or a medkit. Or ordering them, you don't even need to go to cargo for that, just order on the PDA, and pick it up in like 15 minutes.

Medbay will gladly give you a kit or two or make you some basic bills they're swimming in supplies roundstart, and unless the round gets extra chaotic they'll be swimming in them through the entire shift and being bored. Plus you give the new players something to do like learning basic chem by asking for libital and such.

**Comment by sippykot** [comment]
so... this improved the game by... making people run over to medbay and shout at them for supplies, that they might not want to give out/just be busy or annoyed by this?

it literally just adds a chore you need to do roundstart

**Comment by Strazyplus** [comment]
Unfortunate downside of communial game dev. 

> I think taking the cost out of your bank account balance as a sort of debt for priced loadout items is worth considering, if we want to re-add this

300 can be very little so a idea I have is everyone has a bonus budget for equipment that supplements however this money doesn't go into your bank account if not all spent, if you go over it siphons from your bank and you can "overdraw" which a negative mood let would be great and maybe have not fines you 100 to 500 credits for overdraft as punishment.

**Comment by Majkl-J** [comment]
> so... this improved the game by... making people run over to medbay and shout at them for supplies, that they might not want to give out/just be busy or annoyed by this?
> 
> it literally just adds a chore you need to do roundstart

TG does just fine without it with people actually asking med for stuff I can't see why we can't

**Comment by sippykot** [comment]
> > so... this improved the game by... making people run over to medbay and shout at them for supplies, that they might not want to give out/just be busy or annoyed by this?
> > it literally just adds a chore you need to do roundstart
> 
> TG does just fine without it with people actually asking med for stuff I can't see why we can't

we are not TG, removing/nerfing shit form here because "TG is fine without it" is not an argument
if you want TG, play on TG, sorry for being blunt but that's a pretty simple solution

this PR literally does nothing but makes people do more busywork/fetching for basic items
you could make the CL with add: made the roundstart slower and more annoying

**Comment by sneksnek** [comment]
> it literally just adds a chore you need to do roundstart

you can boil a lot of things down to 'chores' like this. it's not a fair representation. players are supposed to engage with mechanics and other players, not loophole around them with loadout items

**Comment by sippykot** [comment]
i dont like it as a med player and i dont like it as a non-med player
this is (another) merge that just nerfs/changes shit and then turns around and tries to make people argue for keeping stuff instead of convincing why removing stuff is good
"90% tg" is not an argument, roundstart busywork is not engaging or interesting

**Comment by sneksnek** [comment]
> i dont like it as a med player and i dont like it as a non-med player this is (another) merge that just nerfs/changes shit and then turns around and tries to make people argue for keeping stuff instead of convincing why removing stuff is good "90% tg" is not an argument, roundstart busywork is not engaging or interesting

this PR already had reasoning propounded in the PR body for the removal of loadout med kits. just read it

if "roundstart busywork" is interacting with other players to get what you need or having to learn mechanics to get it yourself (there are a bunch of ways to get healing outside medical), then i think you're playing the wrong game

**Comment by ReturnToZender** [comment]
I walked up to the chemfridge in front of medical and grabbed some patches they made, this empty medikit means I can finally have useful medicine and pens I can properly refill.

My wallet has never been happier

**Comment by YakumoChen** [comment]
> PR that removes pressing a button in roundstart to sidestep the content and interaction with multiple jobs

> Primary people complaining are the people who take the free sidestep

Nothing is worse as a medical player as having nothing to do because everyone who matters doesn't need your job because they already have supplies

**Comment by NanoCats** [comment]
Also I wish that we could've kept the medpen pouch but making an empty version probably would've been a burden, ough.
